.github/workflows/HcPkgPreReleaseGitops.yaml

```diff
@@ -40,13 +40,13 @@ jobs:
          git update-ref 'refs/heads/${{inputs.releaseTrackingBranch}}' $sha 
          git push origin ${{inputs.releaseTrackingBranch}}
 
-         echo "point qa-test branch to current master"
+         echo "Point qa-test branch to current master"
          git reset HEAD --hard
          git checkout ${{inputs.main}} 
          sha=$(git rev-parse  HEAD)
          echo "master branch sha: $sha"
-         git checkout ${{inputs.qatestBranch}}
-         git update-ref  'refs/heads/${{inputs.qatestBranch}}' $sha 
-         git push origin ${{inputs.qatestBranch}}
+         git checkout ${{inputs.releaseTrackingBranch}}
+         git update-ref  'refs/heads/${{inputs.releaseTrackingBranch}}' $sha 
+         git push  origin ${{inputs.releaseTrackingBranch}} 
```

Comment:
It seems like these lines changed to affect a different branch. Was this intentional?